app/models/event.rb

```diff
@@ -11,6 +11,9 @@ class Event < ActiveRecord::Base
   validates :end_time, presence: true
   validate :start_time_should_be_before_end_time
 
+  scope :before_now, -> { where("start_time < ?", Time.zone.now()) }
+  scope :after_now, -> { where("start_time >= ?", Time.zone.now()) }
```

Comment:
__[rubocop]__
* Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments. [:ref](https://github.com/bbatsov/ruby-style-guide#no-args-no-parens)